src/foreign/vrt.h

```diff
@@ -0,0 +1,377 @@
+/*-
```

Comment:
We definitely don't want to include vrt.h - could we slim down the VSA bits ever so slightly to not depend on this?